src/calendar-settings.js

```diff
@@ -19,6 +19,11 @@ define(["qlik"], function (qlik) {
             });
         return fieldListPromise;
     }
+    function isQlikCloud(){
```

Comment:
This function is used in three separate files, consider moving it to a common place and share it across files instead.